Rules/CommonScripts/ChatCommands/UtilityCommands.as

```diff
@@ -0,0 +1,110 @@
+#include "ChatCommand.as"
+
+class AllArrowsCommand : ChatCommand
+{
+	AllArrowsCommand()
+	{
+		super("allarrows", "Spawn all types of arrows");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
+
+		CBlob@ blob = player.getBlob();
+		if (blob !is null)
+		{
+			Vec2f pos = blob.getPosition();
+			server_CreateBlob("mat_arrows", -1, pos);
```

Comment:
Also affected by the blob spawning comment (posting a comment for each of them for tracking)